dist/htmlminifier.js

```diff
@@ -851,6 +851,9 @@
     else if (attrValue && options.customAttrCollapse && options.customAttrCollapse.test(attrName)) {
       attrValue = attrValue.replace(/\n+|\r+|\s{2,}/g, '');
     }
+    else if (tag === 'script' && attrName === 'type') {
+      attrValue = attrValue.split(/;/g).map(trimWhitespace).join(';');
```

Comment:
I guess you mean `trimWhitespace(attrValue.split(/\s*;\s*/g))`, but okay :stuck_out_tongue: